include/printers.h

```diff
@@ -10,4 +10,11 @@
 
 #ifndef PRINTER_TYPE
     #error "macro PRINTER_TYPE not defined"
-#endif
+    #define PRINTER_MODEL "Not defined"
+#else
+    #if (PRINTER_TYPE == PRINTER_PRUSA_MINI)
+        #define PRINTER_MODEL "MINI"
+    #else
+        #define PRINTER_MODEL "Unknown"
+    #endif // (PRINTER_TYPE == PRINTER_PRUSA_MINI)
+#endif     // PRINTER_TYPE
```

Comment:
It took me some time to fully comprehend what is going on here :D, but I think we can simplify it to something like this

```
#if defined(PRINTER_TYPE) && PRINTER_TYPE == PRINTER_PRUSA_MINI
  #define PRINTER_MODEL "MINI"
#else
  #error "Unknown printer type"
#endif
```

Reply:
commit `f0485ef`